AppInstaller/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.samples.appinstaller">
+
+    <!-- We use this permission to install apps -->
+    <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES" />
+    <!-- We use this permission to update apps in the background -->
+    <uses-permission android:name="android.permission.UPDATE_PACKAGES_WITHOUT_USER_ACTION" />
+    <!-- We use this permission to uninstall apps -->
+    <uses-permission android:name="android.permission.REQUEST_DELETE_PACKAGES" />
+    <!-- We use this permission to get the list of our installed apps -->
```

Comment:
nit: Technically an installer can see all of the packages it has already installed. This permission lets the installer see packages it hasn't yet installed (but may be asked by the user to install). So while this is a permission that is useful to installers of many apps, perhaps we add some clarity to the comment here to be clear that it might not be necessary for all installers. "We use this permission to get information about apps we may try to install before we have installed them."